src/main/java/com/bieganski/ox/Board.java

```diff
@@ -0,0 +1,49 @@
+package com.bieganski.ox;
+
+import java.util.TreeSet;
+
+class Board {
+    private int size;
+    private TreeSet<Field> fields;
+
+    Board(int size) {
+        fields = new TreeSet<>();
+        this.size = size;
+    }
+
+    void addFiled(Field field) throws FieldIsNotEmptyException {
+        if (fields.contains(field))
+            throw new FieldIsNotEmptyException("You can't add existing field");
+        fields.add(field);
+    }
+
+    @Override
+    public String toString() {
+        StringBuilder stringBuilder = new StringBuilder();
+        Field field = fields.first();
+        for (int y = 0; y < size; y++) {
+            for (int x = 0; x < size; x++) {
+                if (checkIsFiledOnGivenCoordinates(field, new Coordinates(x + 1, y + 1))) {
```

Comment:
To be filed means to be written or recorded or put in records. So, if I am writing in some system, that today at that hour I have done this code review, that is filed. That usually is about putting things "in files". So, it's a peculiar choice of words here as well.